src/playdate/domain/lcdbitmap.nim

```diff
@@ -0,0 +1,41 @@
+import std/importutils
+import bindings/[api, types, graphics]
+import bindings/graphics
```

Comment:
for playdate.graphics.freebitmap. It feels like a violation of separation of concerns to import api bindings here. But Nim forces my hand because the `=destroy` must be defined in the same file as the object itself